mvp-spec/dsp-api.md

```diff
@@ -353,10 +432,12 @@ OpenRTB bid request can contain multiple ads - named "impression" in OpenRTB
 specifications. Therefore, a bid request can have multiple Transmission
 Requests. Those Transmissions are added in the "ext" object of each "imp" (for 
 impression) object of the Bid Request. This new object dedicated to the 
-Transmission is named "prebid_sso_transmission".
+Transmission is named "prebid_sso".
```

Comment:
We keep using "Prebid SSO" instead of "PAF" (or equivalent) in all our documents, and I think this is fine for the moment.
However here, this is a technical key used in the data model, so I believe it's important we make sure this is the right name.

And... I'm thinking it's probably not. Shouldn't it be `paf` instead?

Reply:
I have updated with `paf` in both this PR and also the one related to the Ad Server implementation.

There are still "Prebid SSO" instance because "PAF Data" seems less descriptive than "Prebid SSO Data". To be discuss to find another word for this.